pipelines/docs/VolumeProvisioning.md

```diff
@@ -0,0 +1,49 @@
+# Volume Provisioning for Kabanero Pipelineruns
+
+Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The build task, which uses Buildah, also requires a volume mount. The minimum required volume size is five Gi.
+
+## Static Persistent Volumes
+
+If you are using an unmanaged cluster, you can set up a persistent volume to be used by the pipelinerun. A simple persistent volume definition is provided in the following example. Update the path and other values in your `pv.yaml` file provided in this repository, to suit your requirements. 
+
+For instance, the following example is configured for local storage. You might edit this file to configure it for NFS, the standard configuration in a production environment.
```

Comment:
Would reword this to:

The example pv.yaml shows how to configure a Persistent Volume using a hostPath on the node.  This is shown just for the ease of creating the simplest Persistent Volume to enable a pipeline run.  It is not the recommended approach to use hostPath in anything beyond a test environment.

Reply:
updated the document with this feedback.